### ISSUE TITLE
CSP can break the injected script

### ISSUE THREAD

**Issue by neroux** [tarampampam/random-user-agent#24]
## Steps to reproduce:
 1. Load a site with a respective [content security policy](https://en.wikipedia.org/wiki/Content_Security_Policy) (e.g. github.com).
 2. Open the console.
 3. Evaluate `navigator.userAgent`.

## Expected behaviour:
The user agent as defined by the extension should be returned.

## Actual behaviour:
The injected script fails to execute with the message

> Refused to execute inline event handler because it violates the following Content Security Policy directive: "script-src assets-cdn.github.com". Either the 'unsafe-inline' keyword, a hash ('sha256-...'), or a nonce ('nonce-...') is required to enable inline execution.

and the actual user agent is returned.

---------

### System information:
**Browser version**:
```
Google Chrome 62.0.3175.3 (Official Build) dev (64-bit)
```

**Extension version**:
```bash
2.1.7
```

**Comment by neroux** [comment]
https://stackoverflow.com/a/9517879 addresses this issue.

The best way to fix it might be to replace the custom event with a temporary script tag. I quickly tested such an approach and it seems to work.

**Comment by tarampampam** [comment]
Please, show pages URIs, where this error happens.

**Comment by neroux** [comment]
There is one example already given in the report -> Github

**Comment by tarampampam** [comment]
Arrr. There was example with **disabled** javascript protection. Anything else we have?

**Comment by neroux** [comment]
Where did you get it from that it was disabled?

Anyhow, have you tried it with the mentioned hosts?

**Comment by neroux** [comment]
@tarampampam Any input on this issue? You can reproduce it, cant you?

**Comment by neroux** [comment]
@tarampampam

**Comment by neroux** [comment]
Fixed with #25

**Comment by neroux** [comment]
Reopening until a version with this fix is publicly available

@tarampampam
